sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizedReceipt.cs

```diff
@@ -58,12 +58,12 @@ internal static FormField<string> ConvertPhoneNumberField(string fieldName, IRea
             return null;
         }
 
-        internal static FormField<int> ConvertIntField(string fieldName, IReadOnlyDictionary<string, FormField> fields)
+        internal static FormField<long> ConvertInt64Field(string fieldName, IReadOnlyDictionary<string, FormField> fields)
```

Comment:
Wasn't sure if I should call this `ConvertInt64Field`, `ConvertLongField` or `ConvertIntegerField`. Went with `Int64` for clarity. It doesn't matter that much since it's an internal method (and it's not used anywhere else, by the way).